packages/client/hmi-client/src/components/responsive-matrix/matrix-grid.fs.glsl.ts

```diff
@@ -0,0 +1,83 @@
+export default `
+#version 300 es
+precision lowp usampler2D;
+precision highp sampler2D;
+precision highp float;
+in vec2 vUvs;
+uniform sampler2D uColor;
+uniform usampler2D uMicroCol;
+uniform usampler2D uMicroRow;
+uniform float uNumCol;
+uniform float uNumRow;
+uniform float uScreenWidth;
+uniform float uScreenHeight;
+uniform float uWorldWidth;
+uniform float uWorldHeight;
+uniform float uViewportWorldWidth;
+uniform float uViewportWorldHeight;
+uniform bool uGridDisplayBorder;
+uniform float uGridColDisplayLim;
+uniform float uGridRowDisplayLim;
+uniform float uGridColDisplayTransition;
+uniform float uGridRowDisplayTransition;
+out vec4 fragColor;
+void main() {
+	float microColLen = float(textureSize(uMicroCol, 0).x);
+	float microRowLen = float(textureSize(uMicroRow, 0).x);
```

Comment:
Maybe
- microColCellSize, and
- microRowCellSize 

Makes reading a little easier to read?  they kind of look like they are referring to the length of an array data structure.

Reply:
they are referring to the length of the array data structure